debian/changelog

```diff
@@ -0,0 +1,11 @@
+openage (0.3.1-1) jessie; urgency=low
+
+  * Bumping version
+
+ -- Jonas Jelten <jj@sft.mx>  Sun, 27 Mar 2016 22:26:17 +0100
+
+openage (0.3.0-1) unstable; urgency=low
+
+  * Initial release
+
+ -- Jonas Jelten <jj@sft.mx>  Sun, 27 Mar 2016 20:39:39 +0200
```

Comment:
The key thing about work is to avoid it.The `debian/changelog` is a place for the name of the one who actually builds and signs the source packages. :)